Userland/Libraries/LibWeb/Streams/UnderlyingSource.cpp

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright (c) 2023, Matthew Olsson <mattco@serenityos.org>
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#include <LibJS/Runtime/VM.h>
+#include <LibWeb/HTML/Scripting/Environments.h>
+#include <LibWeb/Streams/UnderlyingSource.h>
+#include <LibWeb/WebIDL/CallbackType.h>
+
+namespace Web::Streams {
+
+static JS::ThrowCompletionOr<JS::Handle<WebIDL::CallbackType>> property_to_callback(JS::VM& vm, JS::Value value, char const* property_name)
```

Comment:
https://github.com/SerenityOS/serenity/blob/master/Documentation/Browser/Patterns.md#comments

> When adding non-standard code for a feature that is otherwise well-specified, it should be marked as such.